documentation/source/user_section/installation/windows.rst

```diff
@@ -86,18 +117,57 @@ Windows Subsystem for Linux (WSL) is available on Windows 10 and it makes it pos
       cd /mnt/c
 
 
+WSL Installation with GUI (Optional)
+************************************
+
+If you would like to use SCT's GUI features, or if you would like to try FSLeyes within the same Ubuntu environment, first complete the previous "Basic Installation" section, then continue on to the steps below.
+
+#. Download and install `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_, a program that makes it possible to run Linux GUI programs installed with WSL.
+
+#. Run the newly installed ``XLaunch`` program, then click the following settings:
+
+   - On the "Display settings" page, click "Next".
+   - On the "Client startup" page, click "Next".
+   - On the "Extra settings" page, check the "Disable access control" box, then click "Next".
+   - Click "Finish", then click "Allow access" when prompted by Windows Firewall.
+   - You should now see the X Server icon running in the bottom-right system tray in your taskbar.
+
+#. Next, run the following commands in Ubuntu, depending on the version of WSL you are using.
+
+   WSL1:
+
+   .. code::
+
+      echo "export DISPLAY=localhost:0.0" >> ~/.bashrc
+      echo "export LIBGL_ALWAYS_INDIRECT=0" >> ~/.bashrc
+      source ~/.bashrc
+
+   WSL2:
+
+   .. code::
+
+      echo "export DISPLAY=$(awk '/nameserver / {print $2; exit}' /etc/resolv.conf 2>/dev/null):0.0" >> ~/.bashrc
+      echo "export LIBGL_ALWAYS_INDIRECT=0" >> ~/.bashrc
+      source ~/.bashrc
+
+#. Finally, run the ``sct_check_dependencies`` command in Ubuntu to verify that matplotlib and PyQt figures can be opened by SCT.
+
+#. Optionally, you can install FSLeyes using the following commands:
+
+   .. code::
+
+      source ${SCT_DIR}/python/etc/profile.d/conda.sh
+      conda create -c conda-forge -p ~/fsleyes_env fsleyes -y
+      sudo ln -s ~/fsleyes_env/bin/fsleyes /usr/local/bin/fsleyes
+
+   These instructions will install FSLeyes into a fresh ``conda`` environment, then create a link to FSLeyes so that you can use the ``fsleyes`` command without having to activate the conda environment each time.
+
+
 Option 2: Install with Docker
```

Comment:
Should we move this section outside of the "windows" section, given that this is OS-agnostic?

Reply:
Thinking about it: given that we are pressed with time, feel free to ignore this comment for now. We can reorganize the Docker section later in time. The important thing for now is to have the WSL instructions.

Reply:
> Thinking about it: given that we are pressed with time, feel free to ignore this comment for now. We can reorganize the Docker section later in time. The important thing for now is to have the WSL instructions.

I agree -- the Docker section needs a lot more work and testing than we likely have time for right now.